Translations-Recommendations-Guide.md

```diff
@@ -0,0 +1,99 @@
+```sh
+git commit -m "Challenges of Basic Javascript translated. Closes #4234" seed/challenges/basic-javascript.json 
+git push origin add/ES-translation-basic-javascript
+```
+*  Afterwards, from the webpage of your fork in github.com, make a *pull request* to the branch called ```staging``` in the main repository (see the [help](https://help.github.com/articles/creating-a-pull-request/)). Delete the part that says "Closes #yyyy" from the title of the pull request, and move it to the description area.  
+
+Other translators will be able to see your changes and make comments, which you can discard, or use for making corrections and pushing them again. Be sure, to add some comments explaining the reason of your rejection or what changes you intend to accept. 
+
+Take into account that this file has been created as we work on the translation itself, so many details have not been standardized yet. A final revision might be necessary in order to accomodate any changes made during the process. For this reason, if the comments you receive about your translation don't reflect a serious problem, you can ask in the comments section of the pull request for your changes to be merged, and make the necessary corrections later.
+
+Another way to collaborate in the translation effort is by verifying the translations of other users and making comments in their pull requests.
+
+
+# 5. General guidelines
+
+* Try not to be too formal yet not too casual, just to keep things friendly.
+* In order to make contents more understandable for native speakers of your target language (think of those who don't speak english), translate as much as you can, try to use a word in english only if it is already widely used in the countries where your target language is spoken.
+
+
+# 6. Glosary
+
+It is efficient if all the translators working on the same language create a glosary showing the translation of words in English employed in FreeCodeCamp's challenges. Sometimes there is more than one way of translating some terms, and regional differences might create confusion (for example, some terms might differ between Spanish from Spain and from Latin America, or between the French language employed in Canada and in France). 
+Be democratic! Choose the most appropriate translation by voting and keep a record of the results. One example of such record can be found here: https://docs.google.com/spreadsheets/d/1c60Sl4MAAsZ7biCPgur7A4aVqhErIfwrE1SulPqbOGo/edit#gid=0
+Use the chat room for discussing the glosary, so no one will miss anything.
+
+
+# 7. If you need some help with the Google Translator Toolkit 
+
+You may find help automating the translation process by employing the Google Translator Toolkit, please see: https://github.com/vtamara/fcc_trad
```

Comment:
The steps are still in spanish at https://github.com/vtamara/fcc_trad/blob/master/README.md